comms/Cargo.toml

```diff
@@ -9,28 +9,29 @@ version = "0.0.2"
 edition = "2018"
 
 [dependencies]
```

Comment:
Sorted dependencies and separated local tari_crates from external dependencies for neatness